src/main/drivers/timer.h

```diff
@@ -107,6 +111,26 @@ typedef struct timerHardware_s {
 #endif
 } timerHardware_t;
 
+#ifdef USE_TIMER_MGMT
+
+typedef uint8_t timerTag_t;
+
+typedef struct timerChannelConfig_s {
```

Comment:
I know it all can be inferred, and protected, however that all uses additional flash space, complicates things (code wise - but does enhance user experience), but it can be added later if it becomes a problem.